### ISSUE TITLE
lb[np.logical_and(lb>0, lb<128)] = 2

### ISSUE THREAD

**Issue by LunarSun** [meteorshowers/RCF-pytorch#45]
why lb[np.logical_and(lb>0, lb<128)] = 2? in data_loader.py { class BSDS_RCFLoader(data.Dataset):/def __getitem__(self, index):}

**Comment by LunarSun** [comment]
I got it .it is used to mask the weight in function.py /def cross_entropy_loss_RCF(prediction, label):
